clang/lib/StaticAnalyzer/Checkers/StreamChecker.cpp

```diff
@@ -510,6 +517,14 @@ class StreamChecker : public Checker<check::PreCall, eval::Call,
   ProgramStateRef ensureFseekWhenceCorrect(SVal WhenceVal, CheckerContext &C,
                                            ProgramStateRef State) const;
 
+  ProgramStateRef ensurePtrNotNull(SVal PtrVal, const Expr *PtrExpr,
+                                   CheckerContext &C, ProgramStateRef State,
+                                   const StringRef PtrDescr) const;
+  ProgramStateRef
+  ensureSizeZeroIfLineNull(SVal LinePtrPtrSVal, SVal SizePtrSVal,
```

Comment:
Name of this function is not accurate, this is a special function used only for `getdelim` and this is better to be included in the name (like at `ensureFseekWhenceCorrect`).